doc/en/user/source/services/wms/basics.rst

```diff
@@ -38,18 +38,15 @@ The major differences between versions 1.1.1 and 1.3.0 are:
 Axis Ordering
 -------------
 
-The WMS 1.3.0 specification mandates that the axis ordering for geographic 
-coordinate systems defined in the EPSG database be *latitude/longitude*, or *y/x*. 
-This is contrary to the fact that most spatial data is usually in 
-*longitude/latitude*, or *x/y*. 
-This requires that the coordinate order in the ``BBOX`` parameter
-be reversed for ``SRS`` values which are geographic coordinate systems.
+The WMS 1.3.0 specification mandates using the axis ordering as defined in the EPSG database. For instance, for EPSG:4326 the axis ordering is *latitude/longitude*, or *north/east*. This is contrary to the fact that most spatial data is usually in *longitude/latitude*, or *east/north*. 
 
-For example, consider the WMS 1.1 request using the WGS84 SRS (EPSG:4326):: 
+The WMS 1.1 always uses *east/north* axis ordering. So when upgrading from WMS 1.1, this requires the coordinate order in the ``BBOX`` parameter to be reversed for ``SRS`` with *north/east* axis ordering.
```

Comment:
Wrong, it depends on the CRS. Many CRSs in the EPSG database have east/north orientation, almost (but not all) projected ones in particular. So for example, you would not switch ordinates if the BBOX is expressed in EPSG:3857 (web mercator) or EPSG:32632 (UTM 32N)

Reply:
Agree, it is not always required, that's why I wrote "for ``SRS`` with *north/east* axis ordering".
Maybe I could write it another way to be clearer about that.
For instance:
"The WMS 1.1 always uses *east/north* axis ordering. So when upgrading from WMS 1.1, if the CRS defines *north/east* axis ordering (e.g. EPSG:4326), the coordinate order in the ``BBOX`` parameter must be reversed."

Reply:
Yes, it's clearer to me that way, thank you!

Reply:
tks